lib/PuppeteerSharp/Session.cs

```diff
@@ -133,6 +136,13 @@ internal void OnMessage(string message)
                     }
                 }
             }
+            else if (obj.method == "Tracing.tracingComplete")
+            {
+                TracingComplete?.Invoke(this, new TracingCompleteEventArgs
+                {
+                    Stream = objAsJObject["params"]["stream"].ToString()
```

Comment:
I think you can use `objAsJObject.Value<string>("params.stream")`

Reply:
I wasn't able to do that. This is cleaner though `objAsJObject["params"].Value<string>("stream")`